frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/state-update/vm-settings-tab-state-update.ts

```diff
@@ -93,11 +94,14 @@ const osUpdater = ({ id, prevState, dispatch, getState }: UpdateOptions) => {
   const os = iGetVmSettingValue(state, id, VMSettingsField.OPERATING_SYSTEM);
   const isWindows = os?.startsWith('win');
 
+  const iUserTemplate = iGetLoadedCommonData(state, id, VMWizardProps.userTemplate);
+  const mountTools = !iUserTemplate || !!iGetTemplateGuestToolsDisk(iUserTemplate);
```

Comment:
the value still should depend on the OS no? Eg. when you are creating a template.

Reply:
fixed